orca-keel/src/main/kotlin/com/netflix/spinnaker/orca/keel/model/SubmitDeliveryConfigBody.kt

```diff
@@ -0,0 +1,6 @@
+package com.netflix.spinnaker.orca.keel.model
+
+data class SubmitDeliveryConfigBody(
```

Comment:
This is no longer used since the data is going in the `metadata` field now, right?

Reply:
arg I could have swore I deleted this! yep, removing.